extensions/modules/expathrepo/src/test/java/org/exist/repo/PackageTriggerTest.java

```diff
@@ -52,7 +53,8 @@ public static void setup() throws PermissionDeniedException, IOException, Trigge
         }
 
         // Load XAR file
-        InputStream resourceAsStream = PackageTriggerTest.class.getResourceAsStream("triggertest-1.0-SNAPSHOT.xar");
+
+        InputStream resourceAsStream = PackageTriggerTest.class.getResourceAsStream(xarFile);
```

Comment:
You should probably close this stream when you are finished with it ;-)

Reply:
yep